README.md

```diff
@@ -254,6 +254,18 @@ spec:
 
 In the above case, `myapp` and `myapp2` containers will be instrumented, `myapp3` will not.
 
+#### Override Service Name
+
+The service name for containers goes in the following priority list
+
+* The `instrumentation.opentelemetry.io/service-name` pod annotation (can be `,` seperated list for multiple container services)
```

Comment:
The order does not look right. Some of the object types have the same priority - deployment, SS, CJ, Job